docs/model_zoo/detection.rst

```diff
@@ -126,6 +126,33 @@ Checkout YOLO demo tutorial here: :ref:`sphx_glr_build_examples_detection_demo_y
    | yolo3_mobilenet1.0_voc [3]_ :gray:`(416x416)`| 75.8  | `shell script <https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/detection/yolo3_mobilenet1.0_voc.sh>`_            | `log <https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/detection/yolo3_mobilenet1.0_voc.log>`_                   |
    +----------------------------------------------+-------+--------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
 
+CenterNet
+---------
+
+CenterNet models are evaluated at 512x512 resolution. mAPs with flipped inference(F) are also reported, however, the models are identical.
+Checkout CenterNet demo tutorial here: :ref:`sphx_glr_build_examples_detection_demo_center_net.py`
+
+Note that ``dcnv2`` indicate that models include Modulated Deformable Convolution (DCNv2) layers, you may need to upgrade MXNet in order to use them.
```

Comment:
Do we need to specify which version of MXNet do they need to upgrade to?

Reply:
It's still a nightly version so I won't include it now, I can wait for a release version of mxnet